xonsh/ast.py

```diff
@@ -92,7 +92,7 @@ class CtxAwareTransformer(NodeTransformer):
     a subprocess.
     """
 
-    def __init__(self, parser):
+    def __init__(self, parser, wrap_subprocs=True):
```

Comment:
I don't know if this makes sense.  This flag basically disables the class from performing any work. Rather it seems that whatever call this (the execer) should simply made to skip this, if desired. Even if this is desired here, I believe that this would be better as part of the kwargs passed to the methods, and not part of the instance state.